### PR TITLE
Revise test cases of lsfd

### PR DESCRIPTION
**Pull request by masatake**
(These are not urgent fixes.)

I run the test cases of lsfd on my Raspberry Pi.
I got some failures. Fortunately, lsfd itself works well.
Revising the test cases is enough to avoid the failures.

### CODE REVIEW

File: tests/ts/lsfd/option-inet
Comment (t-8ch) [inline tests/ts/lsfd/option-inet:1]: This test fails on s390x qemu-user.

Comment (masatake) [reply tests/ts/lsfd/option-inet:1]: [491e37a](https://github.com/util-linux/util-linux/pull/2167/commits/491e37a223e9725bdef3bcbbb1ee91261ddea43e) introduces the new failure.

Comment (t-8ch) [reply tests/ts/lsfd/option-inet:1]: Maybe it's easier to split that commit and the fix for the test into its own PR.

Comment (masatake) [reply tests/ts/lsfd/option-inet:1]: I see.

**Pull request merged**